e2e-tests/playwright/e2e/plugins/quay/quay.spec.ts

```diff
@@ -1,28 +1,50 @@
-import { test } from '@playwright/test';
+import { expect, test } from '@playwright/test';
 import { UIhelper } from '../../../utils/UIhelper';
 import { Common } from '../../../utils/Common';
 import { ImageRegistry } from '../../../utils/quay/quay';
 
 test.describe('Test Quay.io plugin', () => {
   const QUAY_REPOSITORY = 'janus-idp/backstage-showcase';
+  let uiHelper: UIhelper;
 
   test.beforeEach(async ({ page }) => {
     const common = new Common(page);
     await common.loginAsGuest();
-  });
 
-  test('Check if Image Registry is present', async ({ page }) => {
-    const uiHelper = new UIhelper(page);
+    uiHelper = new UIhelper(page);
     await uiHelper.openSidebar('Catalog');
     await uiHelper.selectMuiBox('Kind', 'Component');
     await uiHelper.clickLink('backstage-janus');
     await uiHelper.clickTab('Image Registry');
+  });
```

Comment:
I think you will need to move those lines to beforeEach right?

Reply:
correct